src/main/java/scheduler/controller/ScheduleController.java

```diff
@@ -426,6 +427,42 @@ public void initialize() {
         });
     }
 
+    private void randomizeSpaces() {
```

Comment:
Does this thread get killed when the scene exits?

Reply:
Thanks; I meant to and forgot - working on it.